model/database/spans.yaml

```diff
@@ -656,9 +656,9 @@ groups:
         requirement_level:
           conditionally_required: If available.
         note: ""  # overriding the base note
-      - ref: az.namespace
+      - ref: azure.resource.namespace
```

Comment:
Alternatives:
- `azure.provider.namespace`
- `azure.resource.provider.namespace` - precise, but too long
- `azure.rp.namespace` - this acronym is likely to be MS-internal thing